lnwallet/chancloser/chancloser_test.go

```diff
@@ -250,8 +275,10 @@ func TestMaxFeeBailOut(t *testing.T) {
 			// instantiate our channel closer.
 			closeCfg := ChanCloseCfg{
 				Channel: &mockChannel{
-					absoluteFee: absoluteFee,
```

Comment:
there is shadowing in this test with `isInitiator`. it's possible that both tests run with it set to `false`

Reply:
Nice catch, fixed!